index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for pino-abstract-transport 0.4.0
+// Project: https://github.com/pinojs/pino-abstract-transport#readme
+// Definitions by: Diyar Oktay <https://github.com/windupbird144>
+
+/// <reference types="node" />
+/// <reference types="duplexify" />
+
+import { Duplexify } from "duplexify";
```

Comment:
since this is a dependency now for building TS, would be useful to mention installing these types if using ts in readme.

Reply:
I made a new commit with instructions to install @types/duplexify and also @types/node for the Stream API